SCANNER_VERSION

```diff
@@ -1 +1 @@
-2.28.x-39-gb748a2651f
+2.28.x-40-g581c726c42
```

Comment:
If you like, we could update to `2.28.x-48-g8129118a61` to benefit from the caching scanner 😄